### ISSUE TITLE
[Feature Request] Filter Output To Disply Linter Errors

### ISSUE THREAD

**Issue by el10savio** [grafana/dashboard-linter#12]
Currently, the dashboard-linter shows an `OK` log with linter errors if any. This makes the output look very sparse, requiring the user to scroll through to find any errors. 

An ideal feature would be a flag `--show-errors` that only shows the linter errors.

Current Output:

```
Checks that each panel uses the templated datasource.                                                                             
[✔️] OK                                                                                                                            
[✔️] OK                                                                                                                            
[❌] Dashboard '<dashboard>', panel '<panel>' does not use templates datasource, uses '<source>'            
[❌] Dashboard '<dashboard>', panel '<panel>' does not use templates datasource, uses '<source>'   
[✔️] OK                                                                                                                            
```  

Proposed Feature's Output:

```
Checks that each panel uses the templated datasource.                                                                             
[❌] Dashboard '<dashboard>', panel '<panel>' does not use templates datasource, uses '<source>'            
[❌] Dashboard '<dashboard>', panel '<panel>' does not use templates datasource, uses '<source>'   
```

**Comment by rgeyer** [comment]
@el10savio this is a planned feature, in addition to grouping checks by dashboard, panel, and target. We'll be sure to ping you here when this gets implemented.

Thanks for the feedback and for using the tool!

**Comment by marians** [comment]
Based on the outpout I see with the current version, this seems resolved. Example:

![image](https://github.com/grafana/dashboard-linter/assets/273727/97b98edf-9ede-4835-b91c-a57079060164)

**Comment by joebowbeer** [comment]
Fixed by #29
